keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -173,6 +179,19 @@ Windows nodes will adhere to Kubernetes version-skew policy (node to control pla
 - [cncf-k8s-conformance thread](https://lists.cncf.io/g/cncf-k8s-conformance/topic/windows_conformance_tests/27913232)
 - [kubernetes/enhancements proposal](https://github.com/kubernetes/features/issues/116)
 
+### API Changes Proposed
+
+#### Add V1.SecurityContext.WindowsOptions.runAsUsername
+
+Linux uses userID (UID) and groupID (GID) which are represented as integer types. User and group names are not canonical - they are just an alias in /etc/groups or /etc/passwd back to UID+GID. Windows uses a larger binary security identifier (SID) which is stored in the Windows Security Access Manager (SAM) database. This database is not shared between the host and containers, or between containers.
+
```

Comment:
what is the implication of the last sentence ? Is it saying that since Windows uses SID, uid/gid is not relevant at all to Windows and only string user and group names matter ?